### ISSUE TITLE
Add an option to scaffold 3.x projects

### ISSUE THREAD

**Issue by bajtos** [strongloop/loopback#2344]
- Add a new option to loopback-workspace's `createFromTemplate` to specify which LoopBack version to use (2.x or 3.x)
 - Add a new prompt to generator-loopback asking the user which LoopBack version to use.

**Comment by jannyHou** [comment]
- [ ] **generator-loopback**

Add a new prompt called `lbVersion` and pass it into [`createFromTemplate`](https://github.com/strongloop/generator-loopback/blob/master/app/index.js#L172)

```
 {
        name: 'lbVersion',
        message: 'Which version of loopback do you want to install?',
        default: 3.x,
        choices: ['2.x', '3.x'],
}
```

**Comment by bajtos** [comment]
@jannyHou your proposal sounds reasonable 👍

**Comment by jannyHou** [comment]
@ssh24 In this feature, I add a prompt in generator-loopback to choose the version of installed loopback. The choice will be passed to loopback-workspace to create the app with specific `loopback` version.

**Verify steps:**

1. Upgrade to the latest strongloop in staging registry. Make sure `generator-loopback` and `loobpack-workspace` included the latest commit.
2. Run slc loopback to create an app
3. Verify that there is a prompt ask for loopback version and the default option is 2.x
4. Verify `package.json` contains loopback 2.x if choose option 2.x
5. Verify `package.json` contains loobpack 3.x if choose option 3.x

**Comment by ssh24** [comment]
`Verified`

1. Change registry to: `npm config set registry http://ci.strongloop.com:4873/`
2. Install the latest commit: `npm install -g strongloop`
3. Create a loopback app: `slc loopback --name APP_NAME`
4. The loopback version prompt should look like:
<img width="572" alt="100616 node - nvm-versions-node-v4 4 3-bin-slc loopback --name loopback-issues-2344 80x24 terminal today at 3 17 08 pm" src="https://cloud.githubusercontent.com/assets/15173354/15976754/b525302a-2f22-11e6-9e9b-0b5cd7723aaa.png">
5. Select version `2.x` and then `package.json` should look like:
<img width="339" alt="package json sublime text today at 3 28 11 pm" src="https://cloud.githubusercontent.com/assets/15173354/15976800/01b3a8b8-2f23-11e6-9e7e-3d5be94471ac.png">
6. Repeat steps 3 and 4 and select version `3.x` and `package.json` should look like:
<img width="312" alt="package json sublime text today at 3 30 31 pm" src="https://cloud.githubusercontent.com/assets/15173354/15976819/1c359e12-2f23-11e6-8b15-53b4746b2776.png">

**Comment by bajtos** [comment]
Published in `loopback-workspace@3.30.0` and `generator-loopback@1.22.0`
